### ISSUE TITLE
Set up "Friendly" LSF users for them to trial the Proposal Site and fix appropriate issues

### ISSUE THREAD

**Issue by WhittyPM** [UserOfficeProject/stfc-user-office-project#170]
The plan for each facility / route is to enable friendly users to have access to the new proposal submission site to trial and provide feedback. LSF are aware of who they would like those users/submitters to be. 

**Plan:** 
- Review deploying software to production over next few weeks
- Reach out to Andy Ward/Dave Clarke who would like to nominate users for the trial. Gather names and contacts. 
- Deploy software to production _****(Milestone: End of August)_**** 
- Set users up start of September. 

**THIS SPRINT GOAL**

**Comment by TickleThePanda** [comment]
Blocked by #145

**Comment by WhittyPM** [comment]
LSF friendly users are as follows: 

1. Professor Martin King (external user - PI) - M.King@rhul.ac.uk (consent to contact provided) 
2. Andy Ward - Link Scientist
3. Laura Zanetti-Domingues - Internal PI
4. Neil Hunt - Internal PI
5. Greg Greetham - Link Scientist
6. Barbara (Barby) Procacci - External User (awaiting contact details/consent)

@TickleThePanda

**Comment by TickleThePanda** [comment]
I've checked - Andy and Greg should already have access.

**Comment by TickleThePanda** [comment]
One thing the planned questionnaires don't have is whether we need to limit the length of text inputs.

Will we be able to ask Andy and Greg what these should be?

**Comment by WhittyPM** [comment]
Good question. I just reviewed the information Andy/Greg originally sent and that is mostly related to Scientific case rather than the questionnaire itself. I will check with the group and see how they wish to proceed. The original questionnaire does have some limits are we just validating the existing ones. @TickleThePanda

**Comment by WhittyPM** [comment]
Andy W came back with some feedback. Proposals team to review these: 

 []Task

**Comment by WhittyPM** [comment]
@vyshnavi1605  Are these no longer issues after your call with Emma? 

- Facility Area field has value (number weeks requested). Used arrows to select weeks and came up with an error message "value must be a positive number" - to note it was a positive number.
 

- When typing the number in instead another error message comes up "can not answer topic because provided value is not satisfying constraint". Emma feels that users are likely to type in number at times.

**Comment by TickleThePanda** [comment]
Closed after discussion in sprint planning. Outstanding issue may be opened as a new issue.
